modules/helm/install.go

```diff
@@ -12,22 +12,25 @@ import (
 
 // Install will install the selected helm chart with the provided options under the given release name. This will fail
 // the test if there is an error.
-func Install(t *testing.T, options *Options, chartDir string, releaseName string) {
-	require.NoError(t, InstallE(t, options, chartDir, releaseName))
+func Install(t *testing.T, options *Options, chart string, releaseName string) {
+	require.NoError(t, InstallE(t, options, chart, releaseName))
 }
 
 // InstallE will install the selected helm chart with the provided options under the given release name.
-func InstallE(t *testing.T, options *Options, chartDir string, releaseName string) error {
-	// First, verify the charts dir exists
-	absChartDir, err := filepath.Abs(chartDir)
-	if err != nil {
-		return errors.WithStackTrace(err)
-	}
-	if !files.FileExists(chartDir) {
-		return errors.WithStackTrace(ChartNotFoundError{chartDir})
+func InstallE(t *testing.T, options *Options, chart string, releaseName string) error {
+	// If the chart refers to a path, convert to absolute path. Otherwise, pass straight through as it may be a remote
+	// chart.
+	if files.FileExists(chart) {
```

Comment:
Could this lead to an ambiguous situation where there's a chart called, say, `mysql` and a local file called `mysql`?

Reply:
That can happen, but probably unlikely. This is because charts are always namespaced on the repo, so `mysql` is actually `stable/mysql` and our charts will be `gruntwork/k8s-service`. So unless you have a `stable` folder checked out locally, and those happen to have charts, you will reference the remote one.